cfgov/cfgov/settings/__init__.py

```diff
@@ -0,0 +1 @@
+from .local import *
```

Comment:
Actually that won't work because the base settings has to import `os` which is needed here so it will cause a circular import.